lib/esi/response.rb

```diff
@@ -11,7 +9,7 @@ class Response
     def initialize(response, call=nil)
       @original_response = response
       @call = call
-      @data = normalize_results
+      @data = MultiJson.load(body || {}, symbolize_keys: true, object_class: OpenStruct) # rescue OpenStruct.new
```

Comment:
I would still prefer calling `normalize_results` just change `normalize_results` to return this instead.